taxi_service/settings.py

```diff
@@ -44,6 +44,9 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "taxi",
+    "crispy_bootstrap4"
```

Comment:
There is a missing comma at the end of the line after 'crispy_bootstrap4'. This will cause a syntax error. Please add a comma to separate the items in the list.